### PR TITLE
removed deps to old generator

### PR DESCRIPTION
**Pull request by cdietrich**
removed deps to old generator
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -17,7 +17,6 @@ Require-Bundle: org.eclipse.xtext.junit4,
  org.eclipse.jdt.launching;bundle-version="3.9.100",
  org.eclipse.xtext.common.types.ui,
  org.eclipse.emf.mwe2.launch;bundle-version="2.9.1";resolution:=optional;x-installation:=greedy,
```
Comment (szarnekow) [inline org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF]: mwe2 launch - shouldn't this be moved to the additional.bundles section, too?

File: org.eclipse.xtext.ui.tests/build.properties [hunk 2]
```diff
@@ -13,4 +13,6 @@ bin.includes = model/generated/,\
                icons/,\
                testfiles/
 src.includes = about.html
+additional.bundles = org.eclipse.emf.mwe2.launch,\
```
Comment (szarnekow) [inline org.eclipse.xtext.ui.tests/build.properties]: Looks like the handling of the mwe2 dependency is not consistent yet.

File: org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF]: might be some tests that use some classes transitively at runtime.
need to check that

File: org.eclipse.xtext.ui.tests/build.properties (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.ui.tests/build.properties]: fixed

**Comment by cdietrich** [comment]
@szarnekow please reexamine

**Pull request merged**